Makefile

```diff
@@ -13,7 +13,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/m88i/nexus-operator:$(VERSION)
```

Comment:
Not sure if we want to change the SDK generated code. I added `OPERATOR_IMG` for this purpose. Have you researched about this? Can we change it? Not sure yet.

If you feel comfortable to change it, please remove `OPERATOR_IMG` variable.

Reply:
Ah my bad, I changed it because `make docker-build` would build with this weird label and `make deploy` wouldn't work properly, but I just saw there is the `operator-build` recipe which does more or less the same and with the behavior I was expecting. I'll change this back, thanks!